src/TodosContext.tsx

```diff
@@ -0,0 +1,66 @@
+import React, { useState } from 'react';
+import { Todo } from './types/Todo';
+import { useLocalStorage } from './hooks/useLocalStorage';
+import { Status } from './types/Status';
+
+type TodosContextType = {
+  todos: Todo[];
+  setTodos: (value: Todo[]) => void;
+  visibleTodos: () => Todo[];
+  isChecked: boolean;
+  setIsChecked: (value: boolean) => void;
+  filter: Status;
+  setFilter: (value: Status) => void;
+};
+
+export const TodosContext = React.createContext<TodosContextType>({
```

Comment:
let's create our own component 
```
export const useTodo = (): TodosContextType => React.useContext(TodosContext)
```
and reuse everywhere it will help us to avoid redundant import